content/browser-testing/lambdatest.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "LambdaTest review"
-date: 2021-12-07T07:00:00+00:00
```

Comment:
This is the original publication date which needs to remain the same. The "last updated" date is automatically generated based on the last Git commit, so usually this should not need changing.